mmcv/ops/pixel_group.py

```diff
@@ -0,0 +1,39 @@
+import torch
+
+from ..utils import ext_loader
+
+ext_module = ext_loader.load_ext('_ext', ['pixel_group'])
+
+
+def pixel_group(score, mask, embedding, kernel_label, kernel_contour,
+                kernel_region_num, distance_threshold):
+    """Group pixels into text instances, which is widely used text detection
+    methods.
+
+    Arguments:
+        score (np.array): The foreground score with size hxw.
+        mask (np.array): The foreground mask with size hxw.
+        embedding (np.array): The emdedding with size hxwxc to
+            distinguish instances.
+        kernel_label (np.array): The instance kernel index with size hxw.
+        kernel_contour (np.array): The kernel contour with size hxw.
+        kernel_region_num (int): The instance kernel region number.
+        distance_threshold (float): The embedding distance threshold between
+            kernel and pixel in one instance.
+
+    Returns:
+        List[List[float]]: The instance coordinate list. Each element consists
+            of averaged confidence, pixel number, and coordinates
+            (x_i, y_i for all pixels) in order.
+    """
+    score_t = torch.from_numpy(score)
```

Comment:
Can we also support tensor input? This could make the API more general and downstream repos could also more easily use it.

Reply:
I will try to do it